### DISCUSSION TITLE
Help with Tailwind

### DISCUSSION THREAD

**Discussion by fpolli** [filamentphp/filament#14275] [Help]
### Package

Panel builder

### Package Version

v3.2.*

### How can we help you?
EDIT:

So I made a custom theme for my Admin panel following the directions in the documentation:

```
 php artisan make:filament-theme
Using npm v10.8.3

   INFO  Filament theme [resources/css/filament/admin/theme.css] and [resources/css/filament/admin/tailwind.config.js] created successfully.  

   INFO  Filament theme [postcss.config.js] created successfully.  

   WARN  Action is required to complete the theme setup:  

  ⇂ First, add a new item to the `input` array of `vite.config.js`: `resources/css/filament/admin/theme.css`  [DID THAT]
  ⇂ Next, register the theme in the admin panel provider using `->viteTheme('resources/css/filament/admin/theme.css')`  [DID THAT]
  ⇂ Finally, run `npm run build` to compile the theme  

```

Did all that and got this response from npm run build:

```
vite v5.4.4 building for production...
transforming (2) resources/js/bootstrap.js
warn - The `content` option in your Tailwind CSS configuration is missing or empty.
warn - Configure your content sources or your generated CSS will be missing styles.
warn - https://tailwindcss.com/docs/content-configuration

```

My resources/css/filament/admin folder has a theme.css file and a tailwind.config.js file in it. The theme.css file is headed as such:

@import '/vendor/filament/filament/resources/css/theme.css';

@config 'tailwind.config.js';

The tailwind.config.js file is:

```
import preset from '../../../../vendor/filament/filament/tailwind.config.preset'

export default {
    presets: [preset],
    content: [
        './app/Filament/**/*.php',
        './resources/views/filament/**/*.blade.php',
        './vendor/filament/**/*.blade.php',
    ],
}

```

Given all that, I am not sure how the error warning that "The `content` option in your Tailwind CSS configuration is missing or empty" can be true.

**Comment by leandrocfe** [comment]
Probably you have `'resources/css/app.css'` in the input array, in `vite.config.js`
Just remove it

**Reply by fpolli** [reply]
I do. Curious as to what that does here.
